examples/meta/src/kernel/gaussian.sg.in

```diff
@@ -0,0 +1,18 @@
+File traindat = csv_file("@SHOGUN_DATA@/fm_train_real.dat")
+File testdat = csv_file("@SHOGUN_DATA@/fm_test_real.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Features feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Kernel gaussian_kernel = kernel("GaussianKernel", log_width=2.00)
```

Comment:
minor, but this could just be `2.0` no?

Reply:
It asked for double, so i added for more precision. 

But it's your call 🙂

Reply:
Done

Reply:
hmm that's not what double precision means! Double precision is usually a 64-bit representation of a floating point type, where as single will be 32-bit. This is unrelated to the number of 0s you put here :)

Reply:
You're right I guess I need to revisit basic data types xD 😁

Reply:
Shall I git squash?

Reply:
No that's fine, can do that when it's merged

Reply:
Thanks for that comment. I read up on double and revised the concepts again.